Sources/TABResourceLoader/Services/NetworkDataResourceService.swift

```diff
@@ -38,17 +38,22 @@ open class NetworkDataResourceService<NetworkDataResource: NetworkResourceType &
   }
   
   /**
-   Designated initilizer for NetworkDataResourceService
+   Designated initializer for NetworkDataResourceService
    
    - returns: An new instance
    */
   public required init() {
```

Comment:
I think we should remove this from the protocol and make the init with session type the required init